api/all/src/test/java/io/opentelemetry/api/logs/LogApiTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.logs;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+
+class LogApiTest {
+
+  @Test
+  void demo() {
```

Comment:
I like the examples here and I think having separate loggerBuilder and eventBuilder is very nice and actually provides the necessary separation, where all the narrower event-specific functionality goes into the eventBuilder.